addons/archetype-settings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+    https://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <profiles>
+        <profile>
+            <id>smarthome-snapshots</id>
+            <repositories>
```

Comment:
Yes there is. In the example [here](http://stackoverflow.com/questions/43513597/whats-this-bug-of-maven3-3-9-while-init-opendaylight-archetecture) and th documentation [here](https://maven.apache.org/settings.html#Repositories) the profiles are uses, too. The mvn command ignores the `repositories` or `repository` tags if not surrounded by a profile section.